Src/Private/Get-AbrFgtForticare.ps1

```diff
@@ -0,0 +1,168 @@
+
+function Get-AbrFgtForticare {
+    <#
+    .SYNOPSIS
+        Used by As Built Report to returns FortiCare settings.
+    .DESCRIPTION
+        Documents the configuration of Fortinet Fortigate in Word/HTML/Text formats using PScribo.
+    .NOTES
+        Version:        0.1.0
+        Author:         Alexis La Goutte
+        Twitter:        @alagoutte
+        Github:         alagoutte
+        Credits:        Iain Brighton (@iainbrighton) - PScribo module
+
+    .LINK
+        https://github.com/AsBuiltReport/AsBuiltReport.Fortinet.Fortigate
+    #>
+    [CmdletBinding()]
+    param (
+
+    )
+
+    begin {
+        Write-PscriboMessage "Discovering Forticare settings information from $System."
+    }
+
+    process {
+
+        $Forticare = (Get-FGTMonitorLicenseStatus).forticare
+
+        Section -Style Heading2 'FortiCare' {
```

Comment:
you need to add an If statement. Section headings should only be shown if the information to report on is available.